Description: Force the use of /var/lib/ooni for the ooni home
Author: Iain R. Learmonth <irl@torproject.org>
Last-Update: 2017-03-30

--- a/ooni/settings.py
+++ b/ooni/settings.py
@@ -287,9 +287,7 @@
         It defaults to /var/lib/ooni, but if that is not writeable we will
         use the ooni_home.
         """
-        if os.access(VAR_LIB_PATH, os.W_OK):
-            return VAR_LIB_PATH
-        return self.ooni_home
+        return "/var/lib/ooni"
 
     @property
     def user_pid_path(self):
